### PR TITLE
Update language server to support microprofile-config.properties

### PR DESCRIPTION
**Pull request by angelozerr**
Update language server to support microprofile-config.properties

See #181

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
This PR provides the same support for microprofile-config.properties than Quarkus application.properties. 

Please don't close the issue #181 because there is some work to do on JDT LS extension to manage properties for Java hover and CodeLens URL

File: package.json [hunk 1]
```diff
@@ -95,6 +107,10 @@
         "language": "java",
         "path": "./snippets/quarkus.json"
       },
+      {
+        "language": "microprofile-properties",
+        "path": "./snippets/properties.json"
```
Comment (fbricon) [inline package.json]: nope, current snippets are quarkus specific.

File: src/extension.ts [hunk 2]
```diff
@@ -125,6 +125,7 @@ function connectToLS(context: ExtensionContext) {
   return requirements.resolveRequirements().then(requirements => {
     const clientOptions: LanguageClientOptions = {
       documentSelector: [
+	    { scheme: 'file', language: 'microprofile-properties' },
```
Comment (fbricon) [inline src/extension.ts]: indentation

File: package.json (hunk 1 above)
Comment (angelozerr) [reply package.json]: @fbricon fixed

File: src/extension.ts (hunk 2 above)
Comment (angelozerr) [reply src/extension.ts]: @fbricon fixed

**Pull request merged**